index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav class="top-bar">
+        <div class="status">
+          <div id="time-container">
+            <img
+              id="time-icon"
+              src="images/Time Style.png"
+              alt="not available!"
+              class="time"
+            />
+          </div>
+          <div id="status-container">
+            <img
+              id="status-icon"
+              src="images/Group 3.png"
+              alt="not available!"
+              class="internet-wifi-charge"
+            />
+          </div>
+        </div>
+        <div class="header">
+          <div class="text-welcome">
+            <p class="welcome">
+              <a href="#">Welcome</a>
+            </p>
+          </div>
+          <div class="humberger-icon-container">
+            <img
+              id="humberger-icon"
+              src="images/Enabled.png"
+              alt="not available"
+            />
+          </div>
+        </div>
+      </nav>
+      <section class="headline">
+        <div class="primary-text">
+          <span>Hey there. I'm Bakhteyari </span>
+          <span id="colored-text">I'm a software developer</span>
```

Comment:
[ ] kindly use a `p ` tag for your paragraph 🤓.

Reply:
ok.